datafusion/core/src/datasource/physical_plan/mod.rs

```diff
@@ -809,345 +810,4 @@ mod tests {
             extensions: None,
         }
     }
-
-    /// Unit tests for `repartition_file_groups()`
-    #[cfg(feature = "parquet")]
```

Comment:
These tests were moved and refactored into datafusion/core/src/datasource/physical_plan/file_groups.rs